provider/aws/aws.go

```diff
@@ -719,6 +728,8 @@ func (p *AWSProvider) newChange(action string, ep *endpoint.Endpoint) (*Route53C
 				Name: aws.String(ep.DNSName),
 			},
 		},
+		sizeBytes:  0,
+		sizeValues: 0,
```

Comment:
That should not be needed as 0 is the default value for all number variables like `int`.